about.html

```diff
@@ -12,57 +12,80 @@
 </head>
 
 <body>
-    <div class="container about-page">
+   <div class="container2">
+      <header class="header">
+          <img src="/img/lambda-black.png">
+          <nav class="nav">
+              <a class="navie" style="text-decoration: none;" href="/index.html">Home</a>
+              <a class="navie">About</a>
+              <a class="navie">Products</a>
```

Comment:
Don't forget the href attributes on your anchor tags. Even though we are not necessarily going anywhere yet, a transpiler like the one React uses would dislike the lack of hrefs. So keep them as empty strings or `"#"`